src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -4931,6 +4931,7 @@ public static partial class Nullable
         public static int Compare<T>(T? n1, T? n2) where T : struct { throw null; }
         public static bool Equals<T>(T? n1, T? n2) where T : struct { throw null; }
         public static System.Type? GetUnderlyingType(System.Type nullableType) { throw null; }
+        public static ref readonly T GetValueRefOrDefaultRef<T>(in T? nullable) where T : struct { throw null; }
```

Comment:
Is there anywhere in dotnet/runtime we should be using this?

Reply:
Mmh most examples I could find were with small primitive values (eg. `int`, `long` and whatnot). I've found this which might potentiall be a good candidate (given that `ValueTask<T>` is large enough), but then again this is within an asynchronous method so it wouldn't work without restructuring those few lines a bit more, which I'm not sure is necessarily worth it:

https://github.com/dotnet/runtime/blob/a5fda753afce966b9d31c76559636eb515bfa05a/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs#L558-L572

Otherwise I'm not really finding that many examples, at least for now 🤔

Reply:
Ok, thanks.  Sounds like we'll start using it in the DllImportGenerator, but we can leave that for later.